### PR TITLE
Babel transforms; uses script to iterate when multiple modules present

### PR DESCRIPTION
**Pull request by jweisbeck**
Since there may be multiple modules, we need to bundle them separately into their own folders, so a require would look like `require(@buildium/angular-elements/dist/popover`, etc

### CODE REVIEW

File: bundle-all.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+const exec = require('child_process').exec;
+
+fs.readdirSync(argv['i']).forEach(dir => {
+    if (!fs.existsSync(`${argv['o']}/${dir}`)) {
+        fs.mkdirSync(`${argv['o']}/${dir}`);
+    }
+
+    exec(`./node_modules/browserify/bin/cmd.js ${argv['i']}/${dir}/index.js -o ${argv['o']}/${dir}/index.js -x jquery`, function(error, stdout, stderr) {
+        if (error) {
+            console.log(error)
```
Comment (digitil) [inline bundle-all.js]: this should fail the build right? we can use `process.exit(1)` and `console.error`

File: dist/popover/index.js [hunk 2]
```diff
@@ -0,0 +1,194 @@
+(function e(t,n,r){function s(o,u){if(!n[o]){if(!t[o]){var a=typeof require=="function"&&require;if(!u&&a)return a(o,!0);if(i)return i(o,!0);var f=new Error("Cannot find module '"+o+"'");throw f.code="MODULE_NOT_FOUND",f}var l=n[o]={exports:{}};t[o][0].call(l.exports,function(e){var n=t[o][1][e];return s(n?n:e)},l,l.exports,e,t,n,r)}return n[o].exports}var i=typeof require=="function"&&require;for(var o=0;o<r.length;o++)s(r[o]);return s})({1:[function(require,module,exports){
```
Comment (digitil) [inline dist/popover/index.js]: we can remove dist/ and lib/ from version control

File: bundle-all.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+const fs = require('fs');
+const path = require('path');
+const exec = require('child_process').exec;
+
+fs.readdirSync(argv['i']).forEach(dir => {
+    if (!fs.existsSync(`${argv['o']}/${dir}`)) {
+        fs.mkdirSync(`${argv['o']}/${dir}`);
+    }
+
+    exec(`./node_modules/browserify/bin/cmd.js ${argv['i']}/${dir}/index.js -o ${argv['o']}/${dir}/index.js -x jquery`, function(error, stdout, stderr) {
```
Comment (digitil) [inline bundle-all.js]: we can `require('browserify')` and use the programmatic api

**Review by digitil — CHANGES_REQUESTED** [review]

File: package.json [hunk 4]
```diff
@@ -1,13 +1,14 @@
...
+  "version": "1.0.0-2",
   "description": "A collection of directives for building angular applications",
   "main": "index.js",
   "scripts": {
     "test": "karma start --single-run",
-    "dist": "browserify index.js -o dist/buildium-angular-ui.js",
-    "docs": "node .build/docs.js",
-    "preversion": "npm run dist && npm run docs"
+    "compile": "babel src --out-dir lib/ --presets=es2015",
+    "dist": "mkdirp dist && browserify lib/**/index.js -o dist/buildium-angular-elements.js",
```
Comment (jweisbeck) [inline package.json]: Does this glob pattern look right?

Comment (digitil) [reply package.json]: we should move index.js to src/index.js

and then this would be lib/index.js

**Review by ryan-pierce-buildium — APPROVED** [review]
looks good

**Pull request merged**